README.md

```diff
@@ -1 +1,33 @@
-# gallery
\ No newline at end of file
+# gallery
+
+Gallery provides Loom's UI for interacting with the entity data model, as well as downloading and visualizing datasets.
+
+**Setup**
+
+1. Install [cassandra](https://docs.datastax.com/en/cassandra/2.0/cassandra/install/installDeb_t.html)
+2. Clone the entire [loom](http://stash.krypt.int/projects/KRYP/repos/loom/browse) superproject
+3. Install dependencies.
```

Comment:
only thought here is that this can be a little more clear because just "install dependencies" doesn't immediately register as `npm install`, though you have it below.

also, I think the steps can be misunderstood. step 1 makes sense. step 2 makes sense, but going from 2 to 3 doesn't, mainly because you don't say that `gallery` lives inside the superproject.

I'd expand on step 3... explicitly state that you have to `cd` into `loom/gallery`, and then install JS/NPM dependencies with `npm install`